test/jdk/java/lang/management/ThreadMXBean/ThreadCpuTime.java

```diff
@@ -228,15 +230,6 @@ public void run() {
                     " CurrentThreadCpuTime = " + time1 +
                     " > ThreadCpuTime = " + time2);
             }
-/*************
- * FIXME: Seems that on Solaris-sparc,
- * It occasionally returns a different current thread user time > thread user time
-            if (utime1 > utime2) {
-                throw new RuntimeException("TEST FAILED: " + getName() +
-                    " CurrentThreadUserTime = " + utime1 +
-                    " > ThreadUserTime = " + utime2);
-            }
-*/
```

Comment:
Shouldn't this be uncommented and this bit of testing restored? It seems the only reason it was commented out is because of Solaris, which we don't need to worry about anymore. Probably best to leave this to a separate PR if you are going to restore it.

Reply:
Would have been happy to test it and bring it back, have looked into it more:

back in jdk5u we have the same commented out code, we have never run this in general testing. 

I think it's redundant, the two calls are equivalent.

  long utime1 = mbean.getCurrentThreadUserTime();

"This is a convenient method for local management use and is equivalent to calling: getThreadUserTime(Thread.currentThread().getId());"

..which is the other time we would be comparing it with:

  long utime2 = mbean.getThreadUserTime(getId()); (in this class that extends Thread).
  
I think we can presume some os-specific quirk that does not affect us today!

Reply:
ok.